src/sections/General/Footer/footer.style.js

```diff
@@ -148,12 +148,6 @@ const FooterWrapper = styled.section`
 			display: none;
 		}
 	}
-
-	@media only screen and (max-width: 1199px) and (min-width: 704px) {
-		.subscribe {
-			margin-top: -7rem;
-		}
-	}
```

Comment:
This doesn't help becaus, if we remove the top margin then there is a big mpty space below the last 2 columns in the footetr for screen width < 1200px.
Secondly, I don't see the issue reproducible anyway on any of the browser as of now. Please share the reproducible video or so.